homeassistant/components/ovos/__init__.py

```diff
@@ -0,0 +1,14 @@
+"""Support for OpenVoiceOS (OVOS) and Neon AI."""
+from homeassistant.const import CONF_HOST, Platform
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import discovery
+from homeassistant.helpers.typing import ConfigType
+
+DOMAIN = "ovos"
+
+
+def setup(hass: HomeAssistant, config: ConfigType) -> bool:
```

Comment:
Since this is a config entry based integration, you need to implement `async_setup_entry`